<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_funcD_Weather</class>
 <widget class="QWidget" name="Widget_funcD_Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>532</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>696</width>
    <height>532</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>532</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Widget_Assist_Backdrop" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <widget class="QLineEdit" name="searchInput">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLabel" name="cityLabel">
      <property name="geometry">
       <rect>
        <x>423</x>
        <y>20</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>城市名</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Widget_Assist_Backdrop" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <widget class="QLabel" name="tempLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>温度</string>
      </property>
     </widget>
     <widget class="QLabel" name="windHumidityLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>湿度风力</string>
      </property>
     </widget>
     <widget class="QLabel" name="dateLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>81</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>天气图标</string>
      </property>
     </widget>
     <widget class="QLabel" name="weatherDescLabel">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>天气描述</string>
      </property>
     </widget>
     <widget class="QLabel" name="moodLabel">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>95</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>提示语</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Widget_Assist_Backdrop" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>120</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <widget class="WeatherChartWidget" name="chartWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>671</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Widget_Assist_Backdrop" name="widget_4" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="weekLabel0">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel0">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel0">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_6" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="weekLabel1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel1">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_8" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="weekLabel2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel2">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="weekLabel3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel3">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_10" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="weekLabel4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel4">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="weekLabel5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel5">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_12" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="weekLabel6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempRangeLabel6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="weatherIconLabel6">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_Assist_Backdrop</class>
   <extends>QWidget</extends>
   <header>widget_assist_backdrop.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeatherChartWidget</class>
   <extends>QWidget</extends>
   <header>weatherchartwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
